packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -19,10 +19,10 @@ const Container = styled(Div)`
       width: fit-content;
       transition: filter ${timings.slow};
       filter: drop-shadow(0rem ${shadowYOffset}rem ${shadowBlur}rem rgba(0,0,0,${shadowOpacity}));
+      position: relative;
+      ${isOpen ? 'z-index: 7;' : 'z-index: 1;'}
```

Comment:
Can make this a bit easier to read 
```suggestion
     z-index:  ${isOpen ? 7 : 1};
```